### PR TITLE
Pushing Notification Feature

### PR DESCRIPTION
**Pull request by ahmedtaeha**
### Description
Adjusted the notification/announcement feature, utilized spring boot, and removed bootstrap and remained using tailwind.

### CODE REVIEW

File: frontend/adapted_strength-app/src/firebase.js
Comment (CaseyCubed) [inline frontend/adapted_strength-app/src/firebase.js:1]: Are we supposed to be using firebase with this?

**Review by CaseyCubed — COMMENTED** [review]
Not sure to approve this yet. I know there were a couple of meetings around using firebase on top of our other tools. I'll defer to other's reviews before approving this one.

**Comment by ahmedtaeha** [comment]
> Not sure to approve this yet. I know there were a couple of meetings around using firebase on top of our other tools. I'll defer to other's reviews before approving this one.

Firebase just Conveys the notification/announcement from front-end to backend on the given device token

**Review by kyperbelt — APPROVED** [review]

**Review by kyperbelt — APPROVED** [review]
LGTM :+1:

**Pull request merged**